README.md

```diff
@@ -52,30 +78,56 @@ metadata:
   namespace: flux-system
 type: Opaque
 stringData:
-  AWS_ACCESS_KEY_ID: ${YOUR_AWS_ACCESS_KEY_ID}
-  AWS_SECRET_ACCESS_KEY: ${YOUR_AWS_SECRET_ACCESS_KEY}
-  AWS_REGION: ${YOUR_AWS_REGION}
+  AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+  AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+  # Add this only if it's required. Optional for IAM roles
+  AWS_SESSION_TOKEN: ${AWS_SESSION_TOKEN}
 EOF
+
 kubectl apply -f ./aws-secrets-tofu.yaml
+
 ```
 
-3. Next, in the `idpbuilder` folder, navigate to `./examples/ref-implementation/backstage/manifests/install.yaml` and add the following lines for catalog location at line 171 in backstage config to deploy terraform backstage templates to backstage:
+7. **Update the Backstage configuration: **
+
+In the idpbuilder folder, navigate to `./examples/ref-implementation/backstage/manifests/install.yaml` and add the following lines for catalog location at line `171` in the Backstage config to deploy Terraform Backstage templates to Backstage:
```

Comment:
How will this work if you dont have `--package-dir examples/ref-implementation \` in the `idpbuilder` command?

Reply:
Agree. will fix it once we get reply from @nabuskey

Reply:
As far as I am aware, this repo depends on the ref implementation. With the recent addition of support for app set, it may be possible to pick what you need. But for now, you need it.

Reply:
I understand we only need backstage/manifests folder. Can we move this backstage folder under terraform-integrations? 

Does it impact other deployments?

Reply:
I don't know TBH. I haven't played with this repo much. Probably?

Reply:
@nabuskey The question from Vara was more on moving `backstage/manifests` from idpbuiler ref implementation to terraform implementation. I dont think it works but WDYT?

Reply:
The integrations are additional components in a idp deployment stack. We have moved everything under https://github.com/cnoe-io/stacks and we should soon remove the reference to the idpBuilder example directory. It will have to install from the remote as discussed here: https://github.com/cnoe-io/stacks/tree/main/terraform-integrations